src/main/resources/bamm/meta-model/2.0.0/aspect-meta-model-shapes.ttl

```diff
@@ -598,15 +635,16 @@ bamm:ValidEntityInstances
 			 $this ?property ?listInstance .
 			 filter ( not exists { ?listInstance rdf:rest*/rdf:first ?instanceValue } ) .
 			 bind( ?property as ?path )
+			 bind( 'ERR_LIST_REQUIRED' as ?code )
 		   }
 		"""
    ] ;
    sh:sparql [
       a sh:SPARQLConstraint ;
-        sh:message "The value for a list Property of the Entity instance contains an entry with an invalid data type." ;
+        sh:message "Value '{?value}' for list Property '{?path}' of the Entity instance '{$this}' has invalid data type." ;
```

Comment:
"Value '{?value}' for list Property '{?path}' of the Entity instance '{$this}' has an invalid data type."